File: samples_generated_code/samples_1/qwen/baseline_code_java/code_row_55.java
import java.io.BufferedReader;
import java.io.FileWriter;
import java.io.IOException;
import java.io.InputStreamReader;

public class GetentPasswd {
    public static void main(String[] args) {
        String username = "yourUsername"; // Replace with the actual username
        String command = "getent passwd " + username;
        StringBuilder output = new StringBuilder();

        try {
            Process process = Runtime.getRuntime().exec(command);
            BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
            String line;

            while ((line = reader.readLine()) != null) {
                output.append(line).append(System.lineSeparator());
            }

            int exitCode = process.waitFor();
            if (exitCode == 0) {
                try (FileWriter fileWriter = new FileWriter("output.txt")) {
                    fileWriter.write(output.toString());
                }
            } else {
                System.err.println("Command execution failed with exit code: " + exitCode);
            }

        } catch (IOException | InterruptedException e) {
            e.printStackTrace();
        }
    }
}

File: samples_generated_code/sample_1/qwen/baseline_code_c/code_row_55.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    FILE *output;
    char command[256];
    const char *username = "your_username_here";

    output = fopen("output.txt", "w");
    if (output == NULL) {
        return 1;
    }

    snprintf(command, sizeof(command), "getent passwd %s", username);
    system(command);

    fclose(output);
    return 0;
}